docs/advanced/api.rst

```diff
@@ -14,12 +14,24 @@ Installation
 Fedora/CentOS
 -------------
 
-To allow remote access to the control agent REST API, and for agent connections,
+To enable Flocker control service.
+
+.. task:: enable_flocker_control
```

Comment:
The installation instructions already use `sphinx-prompt`, so I'm not going to consider this a blocker for this issue.